### PR TITLE
Add a graph to Contributing.md

### PR DESCRIPTION
**Pull request by hewigovens**
Preview:https://github.com/TrustWallet/wallet-core/blob/xt/docs-graph/docs/Contributing.md

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -54,9 +60,7 @@ Use the `bootstrap.sh` script in the root folder to quickly build and test. Afte
 * Android: `tools/android-test`
 * iOS: `tools/ios-test`
 
-How to generate a Xcode project:
-
-`cmake -Bxcode -GXcode -DCMAKE_BUILD_TYPE=Debug`
+Or use XCode for testing and debugging. Run `cmake -Bxcode -GXcode -DCMAKE_BUILD_TYPE=Debug` to generate an Xcode project. Then go to `swift/` folder, run `pod install` and open `TrustWalletCore.xcworkspace`.
```
Comment (alejandro-isaza) [inline docs/Contributing.md]: Actually you need to go to `xcode` folder, no need for pod install. That is for running the Swift code.

File: docs/Contributing.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 
 Wallet Core implements the cryptographic functionality of blockchains. This includes elliptic curve cryptography, hashing, address derivation and transaction signing. However it *does not* implement other aspects like networking and UI. Wallet core behaves like a black box for higher level users like Trust Wallet; it takes inputs from the blockchain and the user (for instance UTXOs, private keys, etc.) and produces an output (like a signed and encoded transaction).
 
+### How does Trust Wallet interact with Wallet Core
+
+Take Ethereum as an example:
+
+<img src="wallet-core.png">
```
Comment (alejandro-isaza) [inline docs/Contributing.md]: Can you render it at double the size? It looks grainy on a Retina display.

Comment (hewigovens) [reply docs/Contributing.md]: how about now?

File: docs/Contributing.md (hunk 1 above)
Comment (hewigovens) [reply docs/Contributing.md]: yes, updated

File: docs/Contributing.md (hunk 2 above)
Comment (alejandro-isaza) [reply docs/Contributing.md]: No, still blurry. Not sure if it's fixable...

**Pull request merged**